### DISCUSSION TITLE
[声明] 背后的不是把手！！！

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1792] [其他]
![mmexport1723476686183](https://github.com/user-attachments/assets/19f22f5f-758f-475a-ab52-d2361bc0da23)
真是的，总有人以为我腰后是用来抓的，这是高空绳降用的啊！

什么？你问为什么前辈没有这个把手？emmm，一般来说前辈的身体素质是不用担心任何自由落体的。而且她直接用三节棍挂在绳索上就可以滑下去了。
![Screenshot_20240812_233951](https://github.com/user-attachments/assets/ee3b3066-a99b-454f-b9d3-f4a570b31066)

> 视频来源：bilibili円学

> 图源：pixiv山田おとなり

**Comment by WORLD-executeME** [comment]
可是真的很像把手

**Comment by DarkRuSama** [comment]
一个东西的发明，与它的使用没有关系。

**Comment by yutong903** [comment]
道理我都懂，可是你背上为什么有个把手【doge】

**Comment by gaojiaobei** [comment]
治安官大人可爱捏🥰

**Comment by qaqotz** [comment]
明白了，背后的是把手

**Comment by ZhuYuan-148** [comment]
不是不是不是把手！到底是谁一直把我当摩托车啊…

**Comment by 834398220** [comment]
可以用与那个就行，原本是什么不在乎

**Reply by ZhuYuan-148** [reply]
这个不能抓口牙！
